config/runtime/fstests.jinja2

```diff
@@ -24,6 +24,21 @@ class Job(BaseJob):
             print('Kernel config error:', e)
             return 'fail'
 
+    def _kernel_build(self, kdir):
+        try:
+            os.chdir(kdir)
+            result = subprocess.run("make -j$(nproc)", check=True, shell=True)
+            if result.returncode == 0:
+                if os.path.exists('arch/x86/boot/bzImage'):
```

Comment:
I really don't think this is necessary, it's a bit like the check for the presence of the `.config` file.  If the command passed then we can return `pass`.  If it passed but there's a problem with the generated image, some tests will fail in the next step.

Reply:
alright, PR is updated and code on the client side is also updated, once we create a new tarball we will config and build the kernel.